### DISCUSSION TITLE
Histoire Nuxt ignores nuxt.config.ts (extends)

### DISCUSSION THREAD

**Discussion by kleinpetr** [histoire-dev/histoire#497] [Q&A]
Hi,

I am playing with Histoire configuration and it seems it has own nuxt config and ignores my root nuxt.config.ts where I've installed necessary modules, such as Uno.css and also I use Nuxt Layers for extending so I would like to use my nuxt.config.ts with Histoire.

Now it seems that I have to keep 2 standalone configs, one for my Nuxt app and second for the Histoire.

Thanks for clarify :)

**Comment by kleinpetr** [comment]
Is there any possibility how to use the base `nuxt.config.ts` within Histoire? @Akryum

**Reply by awacode21** [reply]
i have the same question @Akryum
